apps/notifier/src/pull-listener.ts

```diff
@@ -15,14 +21,81 @@ export default async function startPullListener(): Promise<void> {
     const subscription = await pubsubClient.getSubscription(pubsubSettings.topicName, pubsubSettings.subscriptionName)
     logger.info('listening messages from topics')
 
+    sgMail.setApiKey(SENDGRID_APY_KEY);
```

Comment:
We should create a proper file for the sendgrid client and add there all the needed logic to manage sendgrid messages. We should please try to divide the code as we'll handle separately the sendgrid part and the nunjucks (or alternative).